core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20210916_clone_workspace_file_transfer.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="dummy" author="mtalbott" id="create_CLONE_WORKSPACE_FILE_TRANSFER">
+        <createTable tableName="CLONE_WORKSPACE_FILE_TRANSFER">
+            <column name="DEST_WORKSPACE_ID" type="BINARY(16)">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="SOURCE_WORKSPACE_ID" type="BINARY(16)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="COPY_FILES_WITH_PREFIX" type="VARCHAR(254)">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+
```

Comment:
If I add the `id` column as a primary key, then we'll need a uniqueness constraint for `DEST_WORKSPACE_ID`, but as it stands it's not necessary since uniqueness is guaranteed by the primary key. No need for a uniqueness constraint on `SOURCE_WORKSPACE_ID` since we should allow a source workspace to be cloned as much as people want. Also no need for the prefix to be unique IMO. The `DEST_WORKSPACE_ID` uniqueness should be sufficient -- a workspace can only be the destination workspace once (ie when it's cloned).